library/modules/bigip_device_auth.py

```diff
@@ -119,6 +119,19 @@
     choices:
       - use-first-server
       - use-all-servers
+  accounting:
```

Comment:
should include version_added:

Reply:
I'm staying away from version added because, what's the answer? Collections 1.0?

Reply:
add the option as requested, the answer to a format is in the code and in our stubber templates:

https://github.com/F5Networks/f5-ansible/blob/devel/devtools/stubs/library_module.py#L21
https://github.com/F5Networks/f5-ansible/blob/devel/library/modules/bigip_cgnat_lsn_pool.py#L21
On a side note I will need to update the template to have 'f5_modules 1.1.0'

Reply:
done.